### ISSUE TITLE
🐛 Program Dashboard not updating when new data is submitted or analysis_id is suppressed

### ISSUE THREAD

**Issue by hknahal** [icgc-argo/platform-ui#2455]
## Description
The Program Dashboard does not update when a new data is uploaded or an analysis_id is suppressed. This issue occurs in across all three environments.

## Examples
1. **Program submits new data but Program Dashboard in PROD does not update to show the new data**
NOTE: The Program Dashboard updates after a few days, so the example reported here is relevant but the donors showed up a week later in the Program Dashboard.
- Data submitter for POG-CA program uploaded data for 10 donors and published the SONG payloads on October 31, 2022 but the Program Dashboard did not update to show the new data.
- Data submitter notified us 4 days later on November 4th (ARGO HelpDesk ticket: https://extsd.oicr.on.ca/browse/ARGOHD-292) that the Program Dashboard had still not updated to reflect the new donor submissions.
-  DCC Admin clicked on program sync button in `DCC Dashboard`, but that did not resolve the issue.
- The following week (week of Nov 7th), the Program Dashboard had updated and the new donors finally show, but not sure why or how.
- The program has encountered this same issue before as well (https://extsd.oicr.on.ca/browse/ARGOHD-282).

2. **Program Dashboard does not update to reflect suppressed analysis_id** (this happens in both QA and DEV)
In QA, the analysis_id "299dbfab-a72f-4bc9-9dbf-aba72f7bc93a" from the DATA-CA program was suppressed: https://submission-song.rdpc-qa.cancercollaboratory.org/studies/DATA-CA/analysis/299dbfab-a72f-4bc9-9dbf-aba72f7bc93a

But the Program Dashboard in QA still shows "1T" under the "Raw Reads" section:
![DO263255](https://user-images.githubusercontent.com/4992044/202553390-74e3dacb-3f35-4939-9c14-0f5f92ed08b1.png)

It should show "0T" since the tumour sample was suppressed. The DEV environment has this same issue as well.

**Comment by hknahal** [comment]
The Mutographs program is experiencing a similar issue where they upload molecular data but the Program Dashboard does not update to reflect their submitted data. Email from Mutographs program (MUTO-INTL) on November 21:

```
We tried to upload 515 normal ESCC bam files to Project MUTO-INTL last week.The bam files appeared to uploaded successfully according to the score client.

However, there appeared to be two issues which we have observed:
        1. We had problems trying to publishing them with the sing client. Some of them were published while others were not “published” on the webpage, even though the sing client returned a successful published response.
        2. We noticed that sometimes we had to give an interval of 5 mins or so between publishing consecutive batches.
```

**Comment by b-f-chan** [comment]
After discussion with Bioinfo during grooming, this issue was identified as a critical blocker for the Mutographs program

**Comment by hknahal** [comment]
Mismatch between Program Dashboard and submission-song:
 
Program Dashboard shows 200 donors with valid Raw Reads (set filter "Raw Reads Status" to "Valid"):

![pog_valid_raw_reads](https://user-images.githubusercontent.com/4992044/204031366-a973049a-74f4-40e7-b09a-43048ae87d8d.png)

But submission-song indicates 202 published donors. The 2 donors that don't show in the Program Dashboard have Normal samples published:

|donorId|submitterDonorId|Tumour/Normal|analysis_id|analysisState|analysisStateHistory updatedAt|
|--|--|--|--|--|--|
|DO252224|18154|Normal|[3e452db7-a356-46d7-852d-b7a35676d773](https://submission-song.rdpc.cancercollaboratory.org/studies/POG-CA/analysis/3e452db7-a356-46d7-852d-b7a35676d773)|PUBLISHED|2022-10-25T21:34:07.157747|
|DO252238|20115|Normal|[46325086-dbc3-48ed-b250-86dbc318eda9](https://submission-song.rdpc.cancercollaboratory.org/studies/POG-CA/analysis/46325086-dbc3-48ed-b250-86dbc318eda9)|PUBLISHED|2022-11-03T15:57:37.704506|

Even without filters in the Program Dashboard, you can't find these two donors in the table. But both donors appear in sample_registration.tsv file as well (downloaded from Program Dashboard "All Clinical Data").

**Comment by Buwujiu** [comment]
Tested in QA DATA-CA with the following steps:
1. published analysis 299dbfab-a72f-4bc9-9dbf-aba72f7bc93a in submission song
2. analysis status changed to `published` 
3. check donor dashboard donor id DO263255 raw reads count became `1T`, which is expected.
4. suppressed the same analysis in sub song
5. analysis status changed to `suppressed`
6. check donor dashboard donor id DO263255 raw reads count became `0T`, also expected.
